### PR TITLE
Fix failure in niriss tweakreg test

### PR DESCRIPTION
**Pull request by mcara**
This PR addresses an issue where a regression test may fail even though the code is working as expected (I think).

**Checklist for PR authors (skip items if you don't have permissions or they are not applicable)**
- [ ] added entry in `CHANGES.rst` within the relevant release section
- [x] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [x] All comments are resolved
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

**Comment by mcara** [comment]
https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1452/

File: jwst/regtest/test_niriss_image.py [hunk 1]
```diff
@@ -54,15 +53,25 @@ def test_niriss_tweakreg_no_sources(rtdata, fitsdiff_default_kwargs):
...
-
-    for fi in result._models:
+    result = datamodels.ModelContainer(
+        rtdata.input,
+        return_open=False,
+        save_open=False
+    )
+
+    for fi in result:
+        fi = fi[:-5] + "_tweakreg" + fi[-5:]
```
Comment (hbushouse) [inline jwst/regtest/test_niriss_image.py:74]:
```suggestion
        fi = fi[:-9] + "_tweakreg" + fi[-9:]
```

**Comment by hbushouse** [comment]
Another regtest run started at https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1456

**Comment by mcara** [comment]
Another try: https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1457/

**Review by hbushouse — APPROVED** [review]
Looks good now. All regtests are passing, including this new one to test the failure condition (step skipped when no sources are found).

**Pull request merged**